server/sonar-db-dao/src/main/java/org/sonar/db/purge/PurgeConfiguration.java

```diff
@@ -30,16 +30,19 @@
 import org.sonar.api.utils.System2;
 import org.sonar.core.config.PurgeConstants;
 
+import static java.util.Arrays.asList;
+import static java.util.Collections.singletonList;
+
 public class PurgeConfiguration {
 
   private final IdUuidPair rootProjectIdUuid;
-  private final String[] scopesWithoutHistoricalData;
+  private final Collection<String> scopesWithoutHistoricalData;
```

Comment:
a Set would be more appropriate than collection. We do care that there is no duplicate values (it doesn't make sense) but indeed order doesn't matter

Reply:
Exact. Anyway this code is going to be dropped soon: https://jira.sonarsource.com/browse/SONAR-10180